### ISSUE TITLE
Text elements should have reasonable contrast ratio

### ISSUE THREAD

**Issue by absorpheus** [maxgerber/cactus#3]
Cactus.io nav link fails WCAG 2.0 

See: 

1. http://leaverou.github.io/contrast-ratio/#%20%2300F-on-dimgrey
2. https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01
